src/routes/login/+page.server.js

```diff
@@ -0,0 +1,40 @@
+import {
+    getFormData,
+    checkIfUsernameExists,
+    checkIfPasswordIsCorrect,
+    getUserId,
+    generateUuid,
+    setSession
+} from '$lib/server/account';
+
+/** @type {import('./$types').Actions} */
+export const actions = {
+    default: async ({ request, locals, cookies }) => {
+        const { username, password } = await getFormData(request);
+
+        const check1 = await checkIfUsernameExists(locals, username);
+        if (!check1) {
```

Comment:
On a pas besoin de la variable `check1`
```suggestion
        if (!await checkIfUsernameExists(locals, username)) {
```